ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionKorrektionsglasensStyrkaMessage.java

```diff
@@ -56,4 +58,9 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(Syn syn) {
```

Comment:
Consider naming it getVisibility(..) instead of set, as the code doesn't set anything.